File: articles/algorithms-in-labor-management-and-organizational-systems/go/workload_burden.go
package main

import "fmt"

func main() {
	pace := 0.84
	hours := 0.72
	fatigue := 0.70
	scheduleVolatility := 0.78
	burden := (pace + hours + fatigue + scheduleVolatility) / 4.0
	fmt.Printf("workload_burden_score=%.4f\n", burden)
}

File: articles/algorithms-in-labor-management-and-organizational-systems/typescript/workload_burden.ts
const pace = 0.84;
const hours = 0.72;
const fatigue = 0.70;
const scheduleVolatility = 0.78;
const burden = (pace + hours + fatigue + scheduleVolatility) / 4.0;
console.log(`workload_burden_score=${burden.toFixed(4)}`);
